recipes/eglibc/eglibc_2.18-r24943.oe

```diff
@@ -3,9 +3,18 @@ require ${PN}.inc
 SRC_REV := "${@d.get('PV').split('-',1)[1]}"
 PV := "${@d.get('PV').split('-',1)[0]}"
 
+RECIPE_FLAGS:machine	+= " eglibc_powerpc_32bit_addr"
```

Comment:
Probably not - I changed when I was having problems with the change since in eglibc it is used as a recipe flags and in c.oeclass it is used as a class flag.

It probably doesn't matter and a single use flag can be used for both?